docs/API.md

```diff
@@ -99,6 +100,28 @@ Make sure that the certificate you build with matches your `fcmSandbox` value.
 
 > Note: The integration between GCM and APNS is a bit finicky. Personally, I feel it is much better to send pushes to Android using GCM and pushes to iOS using APNS which this plugin does support.
 
+#### iOS VoIP Notifications
+
+It is possible to receive VoIP Notifications in iOS that can execute the "notification" event also when the application is in background or closed.
+
+This type of notifications consist only of payload data, so the developer is the responsible of handling the event and do whatever the aplication should do when receiving one of them. The cordova-plugin-local-notifications is a good complement for the VoIP feature.
```

Comment:
First some spelling changes:

```
This type of notifications consists only of payload data, so the developer is the responsible for handling of the event and doing whatever the application should do when receiving one of them. The cordova-plugin-local-notifications is a good complement for the VoIP feature.
```

Second, have you tested this plugin with `cordova-plugin-local-notifications `? There were reports in the past of the two plugins stepping on each others methods.

Reply:
Yes, I have tested it in various phones and it works with the plugin cordova-plugin-local-notifications. My motivation to do this patch was to make phonegap-plugin-push work along with the cordova-plugin-local-notifications in my current application by using the first one to receive push notifications silently and the second one to handle them.